requirements.txt

```diff
@@ -35,6 +36,9 @@ PyGithub
 pytest==2.9.2
 python-bugzilla>=1.2.0
 python-dateutil
+python-keystoneclient
+python-novaclient<7.0
+python-heatclient
```

Comment:
Sure, I was just glancing at it and this caught my attention. This might cause problems in future.